src/walletConnect/screens/Logos.tsx

```diff
@@ -29,7 +34,11 @@ export default function Logos({ dappImageUrl, dappName }: Props) {
         </View>
       </View>
       <View style={styles.logoShadow}>
-        {dappImageUrl ? (
+        {isDappImageSvg ? (
+          <View style={styles.logoBackground}>
+            <SvgUri width={24} height={24} uri={dappImageUrl} />
```

Comment:
This works well with the Curve example, 
but if the SVG is a square image with a solid background, it won't be pretty, right?

Reply:
the svg is smaller in size (24) than the containing circle, so even with a colored background the svg should be contained in the circle. i checked with an svg like https://www.svgrepo.com/show/521955/arrow-down-square.svg and it seems okay to me?

![Simulator Screenshot - iPhone 14 Pro - 2024-01-11 at 10 16 39](https://github.com/valora-inc/wallet/assets/20150449/9af628a1-47c0-4d38-90ac-0f2f6a7a611c)

Reply:
It's ok, best we can do with the current design I think.